tests/order.py

```diff
@@ -79,6 +91,26 @@ def test_remove_product_from_order(self):
         self.assertEqual(json_response["size"], 0)
         self.assertEqual(len(json_response["lineitems"]), 0)
 
-    # TODO: Complete order by adding payment type
+    def test_add_payment_type_to_order(self):
+        """
+        Ensure we can add payment type to order.
+        """
+        self.test_add_product_to_order()
+        
+        url = "/orders/1"
+        paymenttype = {"payment_type_id": 1}
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
+        response = self.client.put(url, paymenttype, format='json')
+        
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+        # Get cart and verify payment type was added
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
+        response = self.client.get(url, None, format='json')
+        json_response = json.loads(response.content)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(json_response["payment_type"], 1)
```

Comment:
I'm getting an assertion error when I run the test: `AssertionError: 'http://testserver/paymenttypes/1' != 1`